apprentice/agents/ModularAgent.py

```diff
@@ -665,6 +696,7 @@ def check(self, state, selection, action, inputs):
             responses = resp['responses']
             for resp in responses:
                 # print(resp['selection'],resp['action'],resp['inputs'], _inputs_equal(resp['inputs'],inputs))
+                agent_logger.error(resp['selection'],resp['action'],resp['inputs'], _inputs_equal(resp['inputs'],inputs))
```

Comment:
I would be careful of logger constructions like this (i.e., that make function calls inside the log call) because those calls will be evaluated whether or not the logger is active. It would be good to gate these behind a check that the appropriate level is active.